tensornetwork/contractors/opt_einsum_paths/path_contractors.py

```diff
@@ -156,12 +231,57 @@ def greedy(net: TensorNetwork,
     The network after full contraction.
   """
   alg = functools.partial(opt_einsum.paths.greedy, memory_limit=memory_limit)
-  return base(net, alg, output_edge_order)
+  return base(nodes, alg, output_edge_order)
 
 
-def auto(net: TensorNetwork,
-         output_edge_order: Sequence[Edge] = None,
-         memory_limit: Optional[int] = None) -> TensorNetwork:
+def _auto_nodes(
```

Comment:
I think you duplicated the code. Please revise the code to avoid the copying and pasting. (If we decide to increase or decrease these cutoffs, we'd have to change it twice and its easy to miss.)